config-builder/build.gradle

```diff
@@ -4,12 +4,14 @@
  * of the MIT license.  See the LICENSE file for details.
  */
 
+description = 'Core library used to create bundles and apply for the CA API Gateway'
+
 dependencies {
     compile 'org.jetbrains:annotations:16.0.2'
-    compile 'commons-io:commons-io:+'
-    compile 'commons-lang:commons-lang:+'
-    compile 'org.apache.commons:commons-collections4:+'
-    compile 'com.google.guava:guava:+'
+    compile 'commons-io:commons-io:2.6'
```

Comment:
fixing versions was a requirement for maven central?

Reply:
yes it was.